packages/e2e-contract-tests/src/contract-ids.json

```diff
@@ -1,4 +1,4 @@
 {
-  "ExternalContract": "0x0f07a23834778a5c1834c11747e615c6ac4272d4bb492797cc2495c4e399edce",
-  "MainContract": "0x8566f14cde46446df6d03971067d60a87abc22f78131f65452d8c5d5bbb33642"
+  "ExternalContract": "0xad6d6050eb465681916c5522a5f326f2addf848c60fc6300c68274c4b1880d12",
+  "MainContract": "0xa80343ddbaf7a2bfc365f89c78e8c3c6123eabae17e4fb33413336ea86c33211"
 }
```

Comment:
this file doesn't need to be commited as they are the IDs from your local network and will never be the same for other person running it locally.

we can even .gitignore it to avoid new commits

Reply:
I'm gonna add to .gitignore